lenra.yml

```diff
@@ -0,0 +1,36 @@
+componentsApi: "1.0"
+generator:
+  docker: ./Dockerfile
+  # dofigen:
```

Comment:
You should remove the commented Dofigen config, shouldn't you ?

Reply:
Yes, i leaved it in case of someone could lead to the same result using dofigen.
I will remove it !